### PR TITLE
Update the running-devnet part of the election tutorial

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
Revised to use steps, some changes to flow, some clarification under using the Kadena client.

### CODE REVIEW

**Comment by Ghislain89** [comment]
You need to run a couple commands:

1. `pnpm turbo format` from the root of the monorepo. This should correctly format one of your changed files.
2. `pnpm run --filter @kadena/hd-wallet format` from the root of the monorepo. This should format one more file. 

Comitting both these changed files, reveals a new build error for kadena docs which you can reproduce locally by running
`pnpm --filter @kadena/docs run build`

The error is: 
` ⨯ brokenlink detected in /src/pages/build/guides/election-dapp-tutorial/02-running-devnet.md (link: /src/pages/github.com/kadena-community/voting-dapp.git election-dapp)
⨯ 1 issues found`

File: packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/02-running-devnet.md [hunk 1]
```diff
@@ -1,235 +1,305 @@
...
+In this tutorial, you'll set up a local development network—`devnet`—to run a blockchain inside of a Docker container on your local computer. 
+You can use this development network to test your smart contracts and experiment with code in an isolated environment that your can reset to a clean state at any time.
+
+## Before you begin
+
+Before you start this tutorial, verify the following basic requirements:
+
+- You have an internet connection and a web browser installed on your local computer.
+- You have a code editor, such as [Visual Studio Code](https://code.visualstudio.com/download), access to an interactive terminal shell, and are generally familiar with using command-line programs.
+- You have cloned the [election-dapp](github.com/kadena-community/voting-dapp.git election-dapp) as described in [Prepare your workspace](/build/guides/election-dapp-tutorial/01-getting-started) and have checkd out the `01-getting-started` branch.
```
Comment (Ghislain89) [inline packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/02-running-devnet.md:30]:
```suggestion
- You have cloned the [election-dapp](https://github.com/kadena-community/voting-dapp.git) as described in [Prepare your workspace](/build/guides/election-dapp-tutorial/01-getting-started) and have checkd out the `01-getting-started` branch.
```

Comment (Ghislain89) [reply packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/02-running-devnet.md:30]: You'll need to update this link in order to make docs build again

**Review by Ghislain89 — DISMISSED** [review]

**Comment by realdreamer** [comment]
Changeset file is missing, adding changeset resolve this issue

File: packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/02-running-devnet.md (hunk 1 above)
Comment (lsgunnlsgunn) [reply packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/02-running-devnet.md]: Thanks for your help, @Ghislain89 !

**Comment by jessevanmuijden** [comment]
Perhaps you could also rename the file `02-running-devnet.md` to make the URL of the page correspond to the title of the page. That may require you to add a redirect from the `[...]/02-running-devnet` URL to the new URL, though.

**Review by jessevanmuijden — APPROVED** [review]

**Review by sstraatemans — APPROVED** [review]

File: packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/02-running-devnet.md (hunk 1 above)
Comment (lsgunnlsgunn) [reply packages/apps/docs/src/pages/build/guides/election-dapp-tutorial/02-running-devnet.md]: fixed the link along with other changes.

**Pull request merged**